task/04-arrays-tasks.js

```diff
@@ -116,7 +138,11 @@ function removeFalsyValues(arr) {
  *    [ 'a', 'b', 'c', 'd', 'e', 'f', 'g' ]  => [ 'A', 'B', 'C', 'D', 'E', 'F', 'G' ]
  */
 function getUpperCaseStrings(arr) {
-   throw new Error('Not implemented');
+   
+    let up = arr.toString().toUpperCase();
+    const ar = up.split(',');
+
+   return ar;
```

Comment:
Please, solve it using only just one map method

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map